examples/app_commands/basic.py

```diff
@@ -77,7 +78,7 @@ async def joined(interaction: discord.Interaction, member: Optional[discord.Memb
 @client.tree.context_menu(name='Show Join Date')
 async def show_join_date(interaction: discord.Interaction, member: discord.Member):
     # The format_dt function formats the date time into a human readable representation in the official client
-    await interaction.response.send_message(f'{member} joined in {discord.utils.format_dt(member.joined_at)}')
+    await interaction.response.send_message(f'{member} joined at {discord.utils.format_dt(member.joined_at)}')
```

Comment:
I think this can go either way.

The resulting string is e.g. "Danny#0007 joined in|at May 16, 2016 13:40"

Reply:
I personally think on goes better than either of these

Reply:
The nuances of the English language: 
"Joined at 13:40" would make sense.
"Joined in 2016" would make sense.
"Joined on May 16" would make sense.

But since this is combined datetime, including only 1 of the `at | in | on` would not as it is only correct for part of the datetime string. 

So I'm thinking thinking to omit the keyword completely, for example, "Danny#007 joined `May 16, 2016 1:40PM`" or like it is now, `at` to match the lib's `joined_at`.

Please let me know what you think.

Reply:
Just omit it since this is reaching excessive bikeshedding territory.